src/main/charts/bitbucket/values.yaml

```diff
@@ -1553,3 +1553,59 @@ openshift:
   # configuration files as ConfigMaps.
   #
   runWithRestrictedSCC: false
+
+# OpenSearch config
```

Comment:
We can refer to our documentation for more details on how to set up OpenSearch properly.

Also, is it necessary to run helm `helm dependency update` after changing `opensearch.enabled` value?